holmes.go

```diff
@@ -152,6 +152,11 @@ func (h *Holmes) startDumpLoop() {
 			continue
 		}
 
+		if h.opts.ReachCPUMax(cpu) {
```

Comment:
I think we can take a common name, because probably ReachCPUMax just one of the reasons to diasble dump pofile.

Reply:
h.Enable?

Reply:
I plan to change its name to `isAbnomal`, what's your opinion?

Reply:
isEnableDump?

Reply:
I have changed its name to EnableDump and it returns an error with specific info.